### PR TITLE
Avoid GCE checking if NO_GCE_CHECK is True

### PR DESCRIPTION
**Pull request by erwanor**
Following issue #93 there is now a variable NO_GCE _CHECK set by default to False that will skip any GCE check if set to True.

### CODE REVIEW

**Comment by craigcitro** [comment]
@anthmgoogle can you take a look?

**Comment by anthmgoogle** [comment]
Looks good to me.

**Comment by aaronwinter** [comment]
Great!
@craigcitro I have signed the contributor license agreement by the way.

**Comment by craigcitro** [comment]
sweet! @aaronwinter you answered my question before i even asked it. `;)`

i'm thinking we should maybe switch from a global to an env var? (or have the global default to the env var?)

**Comment by aaronwinter** [comment]
@craigcitro what do you think of ac6142310a9edc47b1c7c6b591b18a7276bcad62 ?

**Comment by aaronwinter** [comment]
@craigcitro I have pulled the recent updates made to insure backward compatibility. I believe that it is ready to be merged if you are OK with that?

File: oauth2client/client.py [hunk 1]
```diff
@@ -970,7 +974,7 @@ def _get_environment(urlopen=None):
     SETTINGS.env_name = 'GAE_PRODUCTION'
   elif server_software.startswith('Development/'):
     SETTINGS.env_name = 'GAE_LOCAL'
-  elif _detect_gce_environment(urlopen=urlopen):
+  elif NO_GCE_CHECK is not 'True' and _detect_gce_environment(urlopen=urlopen):
```
Comment (craigcitro) [inline oauth2client/client.py]: for strings, we want to use `==` instead of `is`.

**Comment by craigcitro** [comment]
one small nit, otherwise looks good.

**Comment by aaronwinter** [comment]
@craigcitro Done! :+1:

**Comment by craigcitro** [comment]
awesome -- thanks @aaronwinter!

**Pull request merged**